src/MaxText/utils/ckpt_conversion/utils/param_mapping.py

```diff
@@ -431,7 +431,7 @@ def GEMMA2_MAXTEXT_TO_HF_PARAM_MAPPING(config, scan_layers=False):
   return mapping
 
 
-def GEMMA2_MAXTEXT_TO_HF_PARAM_HOOK_FN(config, scan_layers=False, saving_to_hf=False):
+def GEMMA2_MAXTEXT_TO_HF_PARAM_HOOK_FN(config, scan_layers=False, layer_cycle_interval=1, saving_to_hf=False):
```

Comment:
Potentially to set this `layer_cycle_interval=1` as default to avoid changes for each model mapping?